Resources/Prototypes/Entities/Objects/Specific/Medical/randompill.yml

```diff
@@ -86,6 +86,8 @@
   - type: RandomFillSolution
     solution: food
     weightedRandomId: RandomFillStrangePill
+  - type: Pill # Frontier
+    random: true # Frontier
```

Comment:
does this really need two comments, just put it on the component and not the data

Reply:
Yes.